src/azure-cli/HISTORY.rst

```diff
@@ -117,6 +117,7 @@ Release History
 
 **Redis**
 
+* Fixing `az redis update` operation to work for caches with RDB/AOF enabled
```

Comment:
Should not edit History in 2.0.77 section.
Please move it under *Release History* section: **Packaging** below  and  **Storage** above

Reply:
Moved.